tests/cloud_test_logic/cloud_test_index.py

```diff
@@ -30,14 +30,14 @@ class CloudTestIndex(str, Enum):
         -> 4) use structured_image_prepro
     """
 
-    unstructured_text = "unstr_txt"
-    unstructured_image = "unstr_img"
-    unstructured_text_custom_prepro = "unstr_txt_custom_prepro"
+    unstructured_text = "pymarqo_unstr_txt"
+    unstructured_image = "pymarqo_unstr_img"
+    unstructured_text_custom_prepro = "pymarqo_unstr_txt_custom_prepro"
```

Comment:
Is this index created during the test? There is a restriction on the length of the index name (32 chars). Note we add 4 chars as unique identifiers in addition to this string

Reply:
This one is actually not used anywhere and it just needs to be renamed.

Reply:
Can you make sure this index can be created correctly and the custom model can be used? Because this is related to custom models which is kind of important.

Reply:
I could add it and add another test but it will force us to have 4 indexes created per test execution. @pandu-k should we do it?

Reply:
custom models are an important use case. Ideally we find a way to make it work with the current indexes, but if we need to add a new index for this to work we can do that too.

Tbh when we fix this, this test should be extended to include private (auth-required) models